packages/client/hmi-client/src/components/operator/tera-operator-placeholder.vue

```diff
@@ -8,32 +8,52 @@
 <script setup lang="ts">
 import { WorkflowOperationTypes } from '@/types/workflow';
 import plants from '@/assets/svg/plants.svg';
-import seed from '@/assets/svg/seed.svg';
-import plantsSmallToBig from '@/assets/svg/plants-small-to-big.svg';
-import plantAndRoot from '@/assets/svg/plant-and-root.svg';
-import plantAndShovel from '@/assets/svg/plant-and-shovel.svg';
-// import clipBamboo from '@/assets/svg/clip-bamboo.svg';
-import pottedPlants from '@/assets/svg/potted-plants.svg';
-// import talkingPlant from '@/assets/svg/talking-plant.svg'
-// import pdfExtract from '@/assets/svg/pdf-extract.svg'
-import trimmingPlant from '@assets/svg/trimming-plant.svg';
+import calibrateDeterministic from '@assets/svg/operator-images/calibrate-deterministic.svg';
+import editModel from '@assets/svg/operator-images/edit-model.svg';
+import calibrateEnsembleProbabilistic from '@assets/svg/operator-images/calibrate-ensemble-probabilistic.svg';
+import model from '@assets/svg/operator-images/model.svg';
+// import calibrateSimulateDeterministic from '@assets/svg/operator-images/calibrate-simulate-deterministic.svg'
+import optimizeModel from '@assets/svg/operator-images/optimize-model.svg';
+import calibrateSimulateProbabilistic from '@assets/svg/operator-images/calibrate-simulate-probabilistic.svg';
+import simulateDeterministic from '@assets/svg/operator-images/simulate-deterministic.svg';
+import code from '@assets/svg/operator-images/code.svg';
+import simulateEnsembleProbabilistic from '@assets/svg/operator-images/simulate-ensemble-probabilistic.svg';
+import configureModel from '@assets/svg/operator-images/configure-model.svg';
+import simulateProbabilistic from '@assets/svg/operator-images/simulate-probabilistic.svg';
+import coupleModels from '@assets/svg/operator-images/couple-models.svg';
+import stratifyModel from '@assets/svg/operator-images/stratify-model.svg';
+import createModelFromCode from '@assets/svg/operator-images/create-model-from-code.svg';
+import transformDataset from '@assets/svg/operator-images/transform-dataset.svg';
+import createModelFromEquation from '@assets/svg/operator-images/create-model-from-equation.svg';
+import transformModel from '@assets/svg/operator-images/transform-model.svg';
+import dataset from '@assets/svg/operator-images/dataset.svg';
+// import validateModelConfiguration from '@assets/svg/operator-images/validate-model-configuration.svg'
+import document from '@assets/svg/operator-images/document.svg';
```

Comment:
Instead of calling items `deterministic` or `probabilistic` it is better to name them per their performer group(s). Internally everywhere we have been naming these xyz-julia and xyz-ciemss respectively. The notions of a deterministic/probabilistic split is incorrect.


Also strictly speaking, we are optimizing a model-configuration, the optimization process does not change the model itself per se.


Also some of your images look identical, I don't know if it would be better to have a single physical file, but import the same file as different semantic instances. Maybe @YohannParis has some preference here.

Reply:
Re some images looking identical... yes, but that is temporary. The main work here was setting up the page so I can easily swap SVG's when we have a better idea of what we want to put in each node. They will all be different eventually.